inputs/manual_input.csv

```diff
@@ -129,12 +145,33 @@ FT fuel transport ship,investment,2035,35000000,EUR,2020,"Assume comparable tank
 Steam methane reforming,FOM,2015,3,%/year,2015,"International Energy Agency (2015): Technology Roadmap Hydrogen and Fuel Cells , table 15.",Large scale SMR facility (150-300 MW).
 Methanol steam reforming,investment,2020,18016.8665097215,EUR/MW_H2,2020,"Niermann et al (2021): 10.1016/j.rser.2020.110171 , table 4.","For high temperature steam reforming plant with a capacity of 200 MW_H2 output (6t/h). Reference plant of 1 MW (30kg_H2/h) costs 150kEUR, scale factor of 0.6 assumed."
 Methanol steam reforming,lifetime,2020,20,years,2020,"Niermann et al (2021): 10.1016/j.rser.2020.110171 , table 4.",
 Methanol steam reforming,FOM,2020,4,%/year,2020,"Niermann et al (2021): 10.1016/j.rser.2020.110171 , table 4.",
+Fischer-Tropsch,investment,2020,788000,EUR/MW_FT,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), table 8: “Reference scenario”.","Well developed technology, no significant learning expected."
+Fischer-Tropsch,lifetime,2020,20,years,2017,"Danish Energy Agency, Technology Data for Renewable Fuels (04/2022), Data sheet “Methanol to Power”.",
+Fischer-Tropsch,FOM,2020,3,%/year,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), section 6.3.2.1.",
+Fischer-Tropsch,investment,2030,677000,EUR/MW_FT,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), table 8: “Reference scenario”.","Well developed technology, no significant learning expected."
+Fischer-Tropsch,lifetime,2030,20,years,2017,"Danish Energy Agency, Technology Data for Renewable Fuels (04/2022), Data sheet “Methanol to Power”.",
+Fischer-Tropsch,FOM,2030,3,%/year,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), section 6.3.2.1.",
+Fischer-Tropsch,investment,2050,500000,EUR/MW_FT,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), table 8: “Reference scenario”.","Well developed technology, no significant learning expected."
+Fischer-Tropsch,lifetime,2050,20,years,2017,"Danish Energy Agency, Technology Data for Renewable Fuels (04/2022), Data sheet “Methanol to Power”.",
+Fischer-Tropsch,FOM,2050,3,%/year,2017,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), section 6.3.2.1.",
```

Comment:
VOM missing?

Reply:
Yes. I don't think there is one mentioned in the source; I'll dig into the references they are basing their work on.

They also state a higher efficiency of (0.799 p.u.) which could be worth adapting? The DEAs low efficiencies are because:
> FT Liquids efficiency increases as hydrogen production efficiency increases with adoption of more efficient technologies. 2020 and 2030 assume alkaline electrolysis, 2040 is based on PEM systems, and 2050 assumes SOEC. Limited improvement in FT synthesis assumed, although jet fuel selectivity may improve over time.

And converge on 75% in 2050. If the hydrogen and efficiency losses for H2 production are considered elsewhere, the higher efficiency should be justified.

Reply:
The hydrogen production efficiency losses are indeed considered separately.

Reply:
I'll update the efficiency then (hydrogen to FTF efficiency). It would not include electricity. (i.e. not energy efficiency of the process).

WRT `VOM`: DEA specifies 5% of investment cost and cites the Agora report I've used for the new values. I cannot find the 5% in the report and am a bit lost now ...

Reply:
Do we need the `VOM`. DEA is very confusing and their sources are as well. I cannot reproduce their numbers based on the method they describe which they claim to have used to calculate the numbers for `VOM`  and `FOM`.

Reply:
ok